composer.json

```diff
@@ -9,21 +9,21 @@
   "homepage": "https://github.com/webonyx/graphql-php",
   "license": "MIT",
   "require": {
-    "php": "^7.4 || ^8.0",
+    "php": "^7.4 || ^8",
```

Comment:
Is there any difference btw?

Reply:
Semantically, no. I like the simplicity and aesthetics.